SU-SWS/saa_victoria_subtheme
# ADAPT-715: Set margin top 0 on headings and subtitle right after a ce…

…ntered image.

# NOT READY

# Summary
- fix double-spacing after images in vertical faux cards, not sure if an update caused the double spacing or what, but it's different than before.

# Review By (Date)
- 9/18 TODAY

# Urgency
- high

# Steps to Test

1. Pull down the branch or test on all the faux-card pages on saauserguide: https://saauserguide.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/cards/rectangle-vertical-21-card and check other faux card pages for spacing.
2. Test on this test site also: https://saasierracamp-test.sites.stanford.edu/family-camp
3. Specifically check that below a vertical image has padding, but the h2, h3, or h4, or subtitle immediately following the image has padding-top: 0;
The spacing should look good and not too much.

See issue image: https://stanfordits.atlassian.net/browse/ADAPT-715

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket https://stanfordits.atlassian.net/browse/ADAPT-715
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
